libs/tokens/src/updaters/TokensListsUpdater/index.ts

```diff
@@ -35,6 +35,8 @@ interface TokensListsUpdaterProps {
   isGeoBlockEnabled: boolean
 }
 
+const ERRORS_TO_IGNORE = /(failed to fetch)|(load failed)/i
```

Comment:
I think this could use some further explanation on the why you ignore these errors

Reply:
As these are also geoblock related, I would include the GEOBLOCK in the name